strictdoc/core/project_config.py

```diff
@@ -748,6 +748,14 @@ def _load_from_dictionary(
             assert section_behavior in ("[SECTION]", "[[SECTION]]")
 
             if "source_nodes" in project_content:
+                if not {
+                    ProjectFeature.REQUIREMENT_TO_SOURCE_TRACEABILITY,
+                    ProjectFeature.SOURCE_FILE_LANGUAGE_PARSERS,
+                }.issubset(project_features):
+                    print(  # noqa: T201
+                        "WARNING: defining source_nodes without enabling REQUIREMENT_TO_SOURCE_TRACEABILITY and "
```

Comment:
I think we want to always do the lowercase `warning:` moving forward. I am considering this with regard to node validation, and since errors are always displayed as `errors:`, it seems reasonable to be doing `warning:` to match the capitalization.

I did use WARNING: for deprecation messages but now in retrospective I think I was simply too wishful/anxious, wanting the users to migrate ASAP not asap 😄 After a cleanup (separate PR, of course, taking a note in #2068), we could make them to be `warning:` as well.